pegasus/sites.v3/code.org/public/playlab.haml

```diff
@@ -0,0 +1,54 @@
+---
+title: Play Lab
+theme: responsive
+social:
+  "twitter:card": photo
+  "twitter:site": "@codeorg"
+  "twitter:url": "http://code.org/playlab"
+  "twitter:image:src" : "https://code.org/images/mc/mc_social.jpg"
+  "twitter:title": "Minecraft Hour of Code"
```

Comment:
This page will have Minecraft social tags for Twitter. Best to remove everything under social.